src/rust/engine/workunit_store/src/lib.rs

```diff
@@ -43,7 +43,7 @@ pub struct WorkUnit {
   pub parent_id: Option<String>,
 }
 
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
Hm... it feels like it would be potentially incredibly verbose to ever accidentally render this (it would basically only be safe in tests). I'm not sure what the rust convention is in that case, but it feels better to maybe not add `Debug` in this case?

Reply:
For sure, it would be quite verbose; but still helpful enough that I used it while "println debugging" this PR.

In general, the guideline I follow is to be liberal in implementing the debug trait so it facilitates troubleshooting for people who like good ol' print debugging. I don't mind pushing it to production as long as the print statements don't get pushed.

If it goes against the conventions we follow here, though; I would be completely happy removing it and adding it locally only whenever I need it.

Reply:
Not a blocker, I suppose. But if someone did actually use this in production code, it might result in rendering `70k workunits * Nbytes_per_workunit` of output to a console.

Reply:
I've removed the `Debug` trait. It made the PR less atomic anyways, so happy to follow your reasoning :smile: